r/R/util.R

```diff
@@ -45,3 +45,37 @@ is_list_of <- function(object, class) {
 }
 
 empty_named_list <- function() structure(list(), .Names = character(0))
+
```

Comment:
This seems like the kind of stuff that would be in `rlang`. Is it not?

Reply:
Not as far as I can see, although I didn't search extensively

Reply:
FYI these are functions I originally wrote for the queryparser package, to work around flaws in the base R `all.vars()` and `all.names()` and to solve for the fact that base R lacks any `all.funs()` function